packages/comet-extras/src/components/toggle/toggle.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
This import should be added to the import on line 3

Reply:
Done